src/hooks/screen/useChatScreenHook.ts

```diff
@@ -15,7 +15,7 @@ import UserSchema from '../../database/schema/UserSchema';
 import useChatUtilsHook from '../core/chat/useChatUtilsHook';
 import useLocalDatabaseHook from '../../database/hooks/useLocalDatabaseHook';
 import useUserAuthHook from '../core/auth/useUserAuthHook';
-import {CHANNEL_TYPE_PERSONAL} from '../../utils/constants';
+import {CHANNEL_TYPE_GROUP, CHANNEL_TYPE_PERSONAL} from '../../utils/constants';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been updated to include `CHANNEL_TYPE_GROUP` along with `CHANNEL_TYPE_PERSONAL`. This change seems necessary for the new logic introduced in line 165 where the channel type is determined based on the `selectedChannel` object's `channelType` property. 
```diff
- import {CHANNEL_TYPE_PERSONAL} from '../../utils/constants';
+ import {CHANNEL_TYPE_GROUP, CHANNEL_TYPE_PERSONAL} from '../../utils/constants';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->